File: Homework/JavaScript-Fundamentals/04. Operators and Expressions/prime-check.js
function solve(args) {
    // PrimeCheck

    var num = +args[0];
    var maxValue = 100;
    var boundary = Math.sqrt(maxValue); // 10

// check for dividers in the range [2, 10]
    var isPrime = (num > 1) &&
        !((num % 2 == 0) && (num != 2)) &&
        !((num % 3 == 0) && (num != 3)) &&
        !((num % 4 == 0) && (num != 4)) &&
        !((num % 5 == 0) && (num != 5)) &&
        !((num % 6 == 0) && (num != 6)) &&
        !((num % 7 == 0) && (num != 7)) &&
        !((num % 8 == 0) && (num != 8)) &&
        !((num % 9 == 0) && (num != 9)) &&
        !((num % boundary == 0) && (num != boundary));

    console.log(isPrime);
}

File: Homework/CSharp-Part-1/03. Operators-and-Expressions/PrimeCheck.cs
using System;

class PrimeCheck
{
    static void Main()
    {
        int number = int.Parse(Console.ReadLine());
        int maxValue = 100;
        int boundary = (int)Math.Sqrt(maxValue); // 10

        // check for dividers in the range [2, 10]  
        bool isPrime = (number > 1) && 
            !((number % 2 == 0) && (number != 2)) &&
            !((number % 3 == 0) && (number != 3)) &&
            !((number % 4 == 0) && (number != 4)) &&
            !((number % 5 == 0) && (number != 5)) &&
            !((number % 6 == 0) && (number != 6)) &&
            !((number % 7 == 0) && (number != 7)) &&
            !((number % 8 == 0) && (number != 8)) &&
            !((number % 9 == 0) && (number != 9)) &&
            !((number % boundary == 0) && (number != boundary));

        Console.WriteLine(isPrime.ToString().ToLower());
    }
}